src/depackers/stb_vorbis.c

```diff
@@ -2200,13 +2438,13 @@ extern void mdct_backward(mdct_lookup *init, float *in, float *out);
 
 mdct_lookup M1,M2;
 
-static void inverse_mdct(float *buffer, int n, vorb *f, int blocktype)
+void inverse_mdct(float *buffer, int n, vorb *f, int blocktype)
```

Comment:
we should keep it `static`

Reply:
This is only enabled if `LIBVORBIS_MDCT` is defined, which it doesn't look like libxmp ever used. The implementation of `inverse_mdct` that libxmp uses is static and defined at 2683.